### ISSUE TITLE
Select text of repo list filter on click

### ISSUE THREAD

**Issue by adamreisnz** [desktop/desktop#2652]
@shiftkey Now that the text in the repo list filter is remembered, it would probably be useful to also auto-select it when you click into the filter (after not having it selected).

This way you can click and type right away, instead of having to delete the existing text or select it with your mouse first.

**Comment by joshaber** [comment]
>This way you can click and type right away, instead of having to delete the existing text or select it with your mouse first.

Since the text is selected with the list opens, you can already click and type right?

**Comment by j-f1** [comment]
Maybe the filter fields should act like a browser URL bar — clicking it selects the text, allowing you to start over. Arrow keys or a second click give you the regular cursor.

**Comment by adamreisnz** [comment]
> Since the text is selected with the list opens, you can already click and type right?
Correct, but the problem is if, for some reason, you open the repo list, then go to another window altogether (maybe to check something), and come back to the app. 

It's a very minor issue so don't worry about it, but I thought it would be a nice enhancement to select the text whenever you click on it just as it's auto selected after you open it.

> Maybe the filter fields should act like a browser URL bar — clicking it selects the text, allowing you to start over. Arrow keys or a second click give you the regular cursor.

Yes that's exactly the behaviour I wanted to suggest.
